assignments/callbacks.js

```diff
@@ -2,29 +2,117 @@ const items = ['Pencil', 'Notebook', 'yo-yo', 'Gum'];
 
 function firstItem(arr, cb) {
   // firstItem passes the first item of the given array to the callback function.
+  
+  const item =cb(arr);
+  console.log(item);
 }
 
+function printItem(items){
+return items[0];;
+}
+
+firstItem(items,printItem)
+
+
+
+
+
 function getLength(arr, cb) {
   // getLength passes the length of the array into the callback.
+  
+  const item = cb(arr);
+  console.log(item);
+}
+
+
+function printLength(items){
+return items.length;;
 }
 
+firstItem(items,printLength)
+
+
+
+
 function last(arr, cb) {
-  // last passes the last item of the array into the callback.
+  // last passes the last item of the array into the callback
+  
+  const item =cb(arr);
+  console.log(item);
 }
 
+function printItem(items){
+return items[items.length-1];;
+}
+
+last(items,printItem)
+
+
+
+
+
 function sumNums(x, y, cb) {
   // sumNums adds two numbers (x, y) and passes the result to the callback.
+  const sum = cb(x,y);
```

Comment:
So, you wrote the callback function to do the adding, which shows you understand how the callback works with the higher-order function.

For this problem, we want to pass `x + y` into the callback.
```
const sum = cb(x + y);
```
This will be the same for the next few problems.

I appreciate what you did here and in the next few problems, though. 👏